engine/services/inference_service.cc

```diff
@@ -283,6 +293,25 @@ InferResult InferenceService::FineTuning(
   return std::make_pair(stt, r);
 }
 
+bool InferenceService::StopInferencing(const std::string& engine_name,
+                                       const std::string& model_id) {
+  CTL_DBG("Stop inferencing");
+  auto engine_result = engine_service_->GetLoadedEngine(engine_name);
+  if (engine_result.has_error()) {
+    LOG_WARN << "Engine is not loaded yet";
+    return false;
+  }
+
+  if (std::holds_alternative<EngineI*>(engine_result.value())) {
```

Comment:
hmm, so we don't support stop inference for remote model at the moment?

Reply:
yeah, will add it in another PR.